include/intersectables/boundingBox.h

```diff
@@ -0,0 +1,41 @@
+#ifndef BOUNDING_BOX_H
+#define BOUNDING_BOX_H
+
+#include "../hitRecord.h"
+#include "../ray.h"
+#include "intersectable.h"
+#include "math/vector3f.h"
+
+#include <vector>
+
+/**
+ * Axis Aligned Bounding Box (AABB)
+ *
+ *                           top-right
+ *         (min_x, max_y)    (max_x, max_y)
+ *               +-----------------+
+ *               |                 |
+ *               |       AABB      |
+ *               |                 |
+ *               |                 |
+ *               +-----------------+
+ *         (min_x, min_y)    (max_x, min_y)
+ *         bottom-left
+ */
+class BoundingBox : public Intersectable {
+ private:
+  const Vector3f bottomLeft;
+  const Vector3f topRight;
+
+ public:
+  BoundingBox static buildFromVectors(const std::vector<Vector3f>& vectors);
```

Comment:
Does this build a BB around the convex hull of all vectors? If yes, document here.